std_msgs/Header header

bool update
# geometry_msgs/Quaternion orientation
float32 yaw
int32 xmin
int32 ymin
int32 xmax
int32 ymax
geometry_msgs/Vector3 raw_pcl_position
geometry_msgs/Vector3 filter_pcl_position